examples/standalone/scripts/idata_asymmetric.js

```diff
@@ -75,7 +75,7 @@ let run = async () => {
 
 run()
 
-function wait () {
```

Comment:
Is it possible to put the `wait` function in a helper/util file if we're using it all over? (can we pass in `EXIT_CONDITION`)?

Reply:
Yes we can do that, let me send a commit with such an utility function. The only comment is that if you grab one of these files to run them isolated you'll need to make sure to also include/copy that dependency file.